subsys/shell/modules/kernel_service.c

```diff
@@ -35,7 +36,7 @@ static int cmd_kernel_uptime(const struct shell *shell,
 	ARG_UNUSED(argc);
 	ARG_UNUSED(argv);
 
-	shell_fprintf(shell, SHELL_NORMAL, "Uptime: %u ms\n",
+	shell_print(shell, "Uptime: %u ms",
```

Comment:
please fix those shell_prints which will now fit in 80 lines.

Reply:
Done